### DISCUSSION TITLE
Get message of "No requests have been detected" when checking GraphQL Network

### DISCUSSION THREAD

**Discussion by jinglu-jlu** [warrenday/graphql-network-inspector#56] [Q&A]
Hi,

I installed the chrome extension (chrome version 99.0.4844.51 on Macbook bigsur). I get the message that "no requests have been detected" when I am in "GraphQL Network" tab although the network tab does show GraphQL query requests.

Any idea?

Thanks,

Jing

**Comment by warrenday** [comment]
This could happen if you are using an unusual format to send requests. Which client library are you using?

Can you try visiting a site like https://medium.com/ which also uses GraphQL and confirm requests show here?
